react-bootstrap/react-bootstrap.d.ts

```diff
@@ -1,6 +1,8 @@
 // Type definitions for react-bootstrap
 // Project: https://github.com/react-bootstrap/react-bootstrap
-// Definitions by: Walker Burgin <https://github.com/walkerburgin>, Vincent Siao <https://github.com/vsiao>, Danilo Barros <https://github.com/danilojrr>
+// Definitions by: Walker Burgin <https://github.com/walkerburgin>,
+// Vincent Siao <https://github.com/vsiao>, Danilo Barros <https://github.com/danilojrr>,
```

Comment:
this needs to be on the same line i believe for the dt-bot to know how to parse it.